lib/vagrant-service-manager/command.rb

```diff
@@ -101,36 +110,55 @@ def print_help(config = {})
         exit config[:exit_status]
       end
 
-      def check_if_a_service_is_running?(service)
+      def service_running?(service)
         command = "systemctl status #{service}"
         with_target_vms(nil, {:single_target=>true}) do |machine|
           return machine.communicate.test(command)
         end
       end
 
-      def print_all_provider_info
-        @env.ui.info I18n.t('servicemanager.commands.env.nil')
+      def execute_status_display(service = nil)
+        if service
+          status = if service_running?(service)
```

Comment:
Line no 121 to 126 is repeated at 132 to 137 and not following Ruby's DRY(don't repeast yourself) principle.
Could we extract it out in method as :
```
def service_status(service)
     return I18n.t('servicemanager.commands.status.status.running') if service_running?(service)
     I18n.t('servicemanager.commands.status.status.stopped')
end
```